CMakeLists.txt

```diff
@@ -602,19 +602,27 @@ if (BUILD_TESTING)
         target_link_libraries(fuzztest PUBLIC ${PROJECT_NAME})
 
         # Set default values for fuzzing if not defined
-        if(NOT DEFINED FUZZ_TIMEOUT_SEC)
+        if(DEFINED ENV{FUZZ_TIMEOUT_SEC})
```

Comment:
Why switch from definitions to environment variables?

Reply:
Mainly because I did not want to include slightly sensitive information like our s3 bucket locations in our repo. Instead I can hide them by configuring them on CodeBuild configuration. `FUZZ_TIMEOUT_SEC` could be defined in buildspec_fuzz_scheduled.yml but I think it's nicer to define it the same place I define other variables.

Reply:
I made the suggested change to keep timeout as definition and instead define it in buildspec_fuzz_scheduled.yml.